power_grid_model_c/power_grid_model/include/power_grid_model/math_solver/math_solver.hpp

```diff
@@ -6,20 +6,19 @@
 #ifndef POWER_GRID_MODEL_MATH_SOLVER_MATH_SOLVER_HPP
 #define POWER_GRID_MODEL_MATH_SOLVER_MATH_SOLVER_HPP
 
-#include <optional>
+#include "iterative_current_pf_solver.hpp"
+#include "iterative_linear_se_solver.hpp"
+#include "linear_pf_solver.hpp"
+#include "newton_raphson_pf_solver.hpp"
+#include "y_bus.hpp"
```

Comment:
@TonyXiang8787 is there any particular reason why this was marked `// clang-format off` in `main`? I prefer the new implementation but want to double-check with you

Reply:
For some reason the compilation failed if we allowed clang-format to reorder those two headers. @bramstoeller and I have investigated a bit but could not find the course. It is now compiling fine. So maybe that issue is gone. Let merge it in the new way.

Reply:
great 😄